plugin/restore/mysql/mysql.go

```diff
@@ -50,9 +52,62 @@ func New(driver *mysql.Driver, instance *mysqlutil.Instance, connCfg db.Connecti
 	}
 }
 
-// replayBinlog restores the database using incremental backup in time range of [config.Start, config.End).
-func replayBinlog(ctx context.Context, config api.BinlogInfo, binlogDir string) error {
-	return fmt.Errorf("Unimplemented")
+// ReplayBinlog replays the binlog about `originDatabase` from `startBinlogInfo.Position` to `targetTs`.
+func (r *Restore) replayBinlog(ctx context.Context, originalDatabase, pitrDatabase string, startBinlogInfo api.BinlogInfo, targetTs int64) error {
+	if err := r.fetchBinlogUpToTargetTs(ctx); err != nil {
+		return err
+	}
+
+	replayBinlogPaths, err := getReplayBinlogPathList(startBinlogInfo, r.binlogDir)
+	if err != nil {
+		return err
+	}
+
+	// Converts the ts to the local time, which is needed by mysqlbinlog's --stop-datetime option.
+	stopTime := time.Unix(targetTs, 0)
+
+	mysqlbinlogArgs := []string{
+		"--disable-log-bin",
+		fmt.Sprintf(`--rewrite-db=%s->%s`, originalDatabase, pitrDatabase),
+		fmt.Sprintf("--database=%s", pitrDatabase),
+		fmt.Sprintf("--start-position=%d", startBinlogInfo.Position),
```

Comment:
Comment about the behavior about --start-position that applies to the first binlog.